service/orchestrator/metrics.json

```diff
@@ -1,128 +1,450 @@
 [
   {
-    "id": "AdminMFAEnabled"
+    "id": "AdminMFAEnabled",
+    "name": "Admin's MFA: Enabled",
+    "description": "This metric is used to assess if MFA for admins is enabled",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "OTPBasedAuthentication",
+          "PasswordBasedAuthentication"
+        ]
+      }
+    }
   },
   {
-    "id": "AdminMixedDuties"
+    "id": "AdminMixedDuties",
+    "name": "Mixed Duties Applied To Admins",
+    "description": "This metric is used to assess if Mixed Duties is applied appropriately",
+    "category": "",
+    "scale": 3,
+    "range": {
+      "minMax": {
+        "min": 0,
+        "max": 1
+      }
+    }
   },
   {
-    "id": "AnomalyDetectionEnabled"
+    "id": "AnomalyDetectionEnabled",
+    "name": "Anomaly Detection: Enabled",
+    "description": "This metric is used to assess if Anomaly Detection is enabled",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
-    "id": "AnomalyDetectionOutput"
+    "id": "AnomalyDetectionOutput",
+    "name": "Anomaly Detection: Output",
+    "description": "This metric is used to assess if the output is ???",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "*"
+        ]
+      }
+    }
   },
   {
-    "id": "AnonymousAuthentication"
+    "id": "AnonymousAuthentication",
+    "name": "Anonymous Authentication",
+    "description": "This metric is used to assess if it is possible to authenticate anonymously",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
     "id": "AtRestEncryptionAlgorithm",
     "name": "Storage Encryption: Algorithm",
     "description": "This metric describes the algorithm of a storage encryption block cipher",
     "category": "CKM-03",
     "scale": 1,
-    "allowedValues": {
-      "values": [
-        "AES128",
-        "AES256",
-        ""
-      ]
+    "range": {
+      "allowedValues": {
+        "values": [
+          "AES128",
+          "AES256",
+          ""
+        ]
+      }
     }
   },
   {
     "id": "AtRestEncryptionEnabled",
     "name": "Storage Encryption: Enabled",
     "category": "CKM-03",
-    "description": "This metric describes whether storage encryption is enabled",
+    "description": "This metric describes if storage encryption is enabled",
```

Comment:
if used with "describes", "whether" is the correct term here, otherwise you are describing something only if storage encryption is enabled and that does not make sense ;)